src/visitors/sympy_conductance_visitor.cpp

```diff
@@ -5,13 +5,18 @@
  * Lesser General Public License. See top-level LICENSE file for details.
  *************************************************************************/
 
+#include "visitors/sympy_conductance_visitor.hpp"
```

Comment:
could you clarify why this is needed at the top? if some type info needed, could before `#include "ast/ast.hpp"` with extra spacing to avoid sorting?

Reply:
I start thinking that every .cpp should begin with the inclusion of its corresponding header. This way, you can ensure that the .hpp is not missing a required header.
If it does, better have a compilation issue in the corresponding .cpp than in the rest of the project (or in another project if the .hpp is public) that includes it in order to use it.

If you agree, we can generalize that to the rest of the project in a future PR. Otherwise, I can move the `#include` to the proper location.

Reply:
> I start thinking that every .cpp should begin with the inclusion of its corresponding header.

I also didn't realise before why this is necessary. It perfectly make sense now (to make sure the header is not dependent on other includes). I see this also documented [here]( https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes).

Reply:
Perfect. Thank you for the link, I was considering adding this to our coding conventions :)